### PR TITLE
rpm test

### PR DESCRIPTION
**Pull request by raphaelgavache**
Test e2e for rpm

### CODE REVIEW

**Review by BaptisteFoy — APPROVED** [review]

**Review by KevinFairise2 — APPROVED** [review]

File: test/new-e2e/tests/updater/linux_test.go [hunk 1]
```diff
@@ -34,13 +35,28 @@ const (
...
-	)))
+		awshost.WithEC2InstanceOptions(ec2.WithOSArch(distro, arch)),
+	)),
+		e2e.WithStackName(testName),
+	)
+}
+
+func TestCentOS(t *testing.T) {
+	t.Parallel()
+	runTest(t, "rpm", os.AMD64Arch, os.CentOSDefault)
```
Comment (pducolin) [inline test/new-e2e/tests/updater/linux_test.go:54]: **🔨 warning**
There is a risk of OOM for the runner, we suggest running one job per Test per remote instance

Comment (raphaelgavache) [reply test/new-e2e/tests/updater/linux_test.go:54]: I'll update on a next PR where I'm adding more cases

File: pkg/updater/service/datadog_agent.go [hunk 2]
```diff
@@ -72,15 +72,16 @@ func SetupAgentUnits() (err error) {
...
 	for _, unit := range stableUnits {
 		if err = startUnit(unit); err != nil {
 			return
 		}
 	}
-	if err = createAgentSymlink(); err != nil {
-		return
-	}
-	err = installinfo.WriteInstallInfo("updater_package", "manual_update_via_apt")
+	err = createAgentSymlink()
```
Comment (pducolin) [inline pkg/updater/service/datadog_agent.go:84]: **❓ question**
Why is this error not handled? Is the linter running on `pkg/updater`? I am surprised this is passing

**Comment by pducolin** [comment]
This PR changes are covered by e2e tests, should have had `qa/done` label

File: pkg/updater/service/datadog_agent.go (hunk 2 above)
Comment (raphaelgavache) [reply pkg/updater/service/datadog_agent.go:84]: the err still passes in the return, thanks to the func signature
`SetupAgentUnits() (err error)`

**Pull request merged**